monitoring.yaml

```diff
@@ -22,9 +22,12 @@ components:
   - repo_name: "tiflow"
     monitor_path: "metrics/grafana"
     rule_path: "metrics/alertmanager"
+  - repo_name: "tiflow"
+    monitor_path: "dm/metrics/grafana"
+    rule_path: "dm/metrics/alertmanager"
+  - repo_name: "tiflow"
+    monitor_path: "dm/metrics/grafana"
+    rule_path: "dm/metrics/alertmanager"
```

Comment:
repeat twice?

Reply:
fixed in https://github.com/pingcap/monitoring/pull/188/commits/0e9dc122d85877ea91a0e17f5ad48d5046f9c4ea